### ISSUE TITLE
Deployment documentation needs to show Seamless mode first, rather then classic "simple"

### ISSUE THREAD

**Issue by tiberiuichim** [plone/volto#4576]
https://6.dev-docs.plone.org/volto/deploying/simple.html

**Comment by stevepiercy** [comment]
That link goes to [Simple deployment](https://6.dev-docs.plone.org/volto/deploying/simple.html). The word "seamless" does not appear on that page.

Do you mean to reorder the items in the navigation under [Deploying](https://6.dev-docs.plone.org/volto/deploying/index.html) instead?

**Comment by Darkcoder011** [comment]
yes this link goes to [Simple deployment](https://6.dev-docs.plone.org/volto/deploying/simple.html). the Word does not appear in that page 

@tiberiuichim

**Comment by tiberiuichim** [comment]
@stevepiercy yes, it should not be first in the navigation, and it should clearly state that is "install Volto on older Plone without access to plone.rest 2.0".

**Comment by stevepiercy** [comment]
@tiberiuichim would you prefer this order?

- [Seamless mode](https://6.dev-docs.plone.org/volto/deploying/seamless-mode.html)
- [Simple deployment](https://6.dev-docs.plone.org/volto/deploying/simple.html)
- [Deployment using a node process manager (PM2)](https://6.dev-docs.plone.org/volto/deploying/pm2.html)
- [Apache](https://6.dev-docs.plone.org/volto/deploying/apache.html)
- [Integration with Sentry](https://6.dev-docs.plone.org/volto/deploying/sentry.html)
- [critical.css (above the fold) optimizations](https://6.dev-docs.plone.org/volto/deploying/performance.html)

Should [Simple deployment](https://6.dev-docs.plone.org/volto/deploying/simple.html) have a deprecation warning at the top, like so? Please provide specific details as needed.

> ### Deprecated
> 
> This installation method requires `plone.restapi` < 2.0.
> It is recommended only for Plone 5.2 and older.
> For Plone 6 and later, use [Seamless mode](https://6.dev-docs.plone.org/volto/deploying/seamless-mode.html).

**Comment by tiberiuichim** [comment]
The whole thing has to be refactored. "Simple deployment" information is relevant, but the https://6.dev-docs.plone.org/volto/deploying/simple.html#reverse-proxies needs to be updated, it should be just a "hey, this is also how you can do it", just a note in the new docs.

I'll try to refactor this today, I may find 30 minutes to do so.

**Comment by aniasusual** [comment]
hi @tiberiuichim, I want to work on this issue if the changes are still required, if you can kindly help me what do I need to amend in reverse-proxies section under "hey, this is also how you can do it" I am happy to contribute.

**Comment by stevepiercy** [comment]
@aniasusual thank you for your interest. But, sorry, this issue was incorrectly tagged as `41 level: easy` (probably at first thought) and is actually `41 level: complex` (now that @tiberiuichim explained the complexity), and requires someone deeply familiar with Volto to refactor the entire section.
